lib/load.php

```diff
@@ -97,6 +96,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // Experimental features.
 remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WP 6.0's stopgap handler for Webfonts API.
+require __DIR__ . '/experimental/behaviors.php';
```

Comment:
Should the comment `Turns off WP 6.0's stopgap handler for Webfonts API` be updated or removed?